hw/xfree86/compat/log.c

```diff
@@ -26,3 +27,19 @@ void xf86Msg(MessageType type, const char *format, ...)
     LogVMessageVerb(type, 1, format, ap);
     va_end(ap);
 }
+
+_X_EXPORT void xf86MsgVerb(MessageType type, int verb, const char *format, ...)
```

Comment:
same here: add comment that it's needed just for the old, not the newer one (use the exact version number) ?